scripts/create_compdb.py

```diff
@@ -57,10 +61,23 @@ def _build_generated_files(bazel: str) -> None:
 
 
 def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Generates compilation data used by clangd.",
+        allow_abbrev=False,
+    )
```

Comment:
I think we usually use the doc string for the script here:

```suggestion
    parser = argparse.ArgumentParser(description=__doc__)
```

Reply:
Sounds good. I left the `allow_abbrev=False,` in (otherwise something like `--also` is matched). Let me know if you want that removed.